README.md

```diff
@@ -193,8 +193,11 @@ $recommendations = $response->getRecommendation()->getData();
 You can also set more granular options of the recommendation command:
 
 ```php
-$recommendation = UserRecommendation::create('user-id', 5, 'test-scenario', 1.0, 3600);
-$recommendation->setFilters(['for_recommendation = 1'])
+$recommendation = UserRecommendation::create('user-id', 'test-scenario')
+    ->setCount(5)
```

Comment:
Is it possible to explain what is the default behavior and what happens if the setters are used? 

I mean something like: 

> Default behavior is specified in Matej admin console, but it could be overridden on per-request basis using `setCount()`, `setRotationRate()` etc. methods of `UserRecommendation`.

Reply:
Ok but I would like to point out that so far the readme didn't mention anything that could be set using admin console and it was expected that the user understands how to use the API. I don't really see how this is different.

Reply:
That's not technically true. They can't be set using admin console. If they are not set, Matej will use some defaults but they are not configurable using the admin console. I assumed the reader implicitly understands that if they don't configure a value manually, Matej uses some default. So in this case I would have to write something like "You can set some values manually and if you don't Matej uses some defaults.". That's quite obvious and also it's not documenting behaviour of this library but behaviour of Matej which should be described in Mate documentation.

Reply:
Btw. this is exactly the same behaviour as before. For example minimalRelevance was always optional argument configured by setters and from the point of view of user of this library used default value if not set.

Reply:
But as someone using the library, I would consider important to know what happens, if I don't specify the values on per-request basis. Whether some default defined in the library will be used, whether this is internal on Matej side etc. What about something like this:

```
You can also set more granular options of the recommendation command and overwrite Matej default behavior on per-request basis:
```

(I thought from our discussion those values would be configurable by users in the admin consele - I probably got it wrong :-) )